packages/bpk-component-content-cards/src/BpkContentCards.module.scss

```diff
@@ -0,0 +1,53 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+@import '~bpk-mixins/index';
+
+$bpk-spacing-v2: true;
+
+.bpk-content-card {
+  &__headerText {
+    margin-bottom: bpk-spacing-base();
+    font-size: $bpk-h-2-font-size;
+    font-weight: $bpk-font-weight-bold;
+    line-height: $bpk-line-height-xxl;
+
+    @include bpk-breakpoint-mobile {
+      font-size: $bpk-font-size-xl;
+      line-height: $bpk-line-height-xl;
```

Comment:
I wonder if these styles are some particular `heading-2`(or `heading-3`) styles https://backpack.github.io/sassdoc/#mixin-bpk-heading-2

Reply:
Yes please use the text styles to set these and not use them directly to set styles as we don't want to be mixing sizes and styles